osu.Game/Overlays/Profile/Sections/PaginatedContainer.cs

```diff
@@ -97,16 +87,133 @@ private void onUserChanged(ValueChangedEvent<User> e)
         {
             VisiblePages = 0;
             ItemsContainer.Clear();
-            ShowMoreButton.Hide();
 
             if (e.NewValue != null)
                 ShowMore();
         }
 
-        protected virtual void ShowMore()
+        protected virtual void ShowMore() => MoreButton.IsLoading = true;
```

Comment:
i think this should be made abstract, and the moreButton action be moved outside of it (to avoid potentially not calling it).